hex/src/lib.rs

```diff
@@ -35,9 +35,9 @@ pub mod buf_encoder;
 pub mod display;
 pub mod parse;
 
-pub use buf_encoder::BufEncoder;
-pub use display::DisplayHex;
-pub use parse::{Error, FromHex};
+pub use buf_encoder::*;
+pub use display::*;
+pub use parse::*;
```

Comment:
In 4baf4410551693e5b0e210744abe69fe2bee1cc5:

I really dislike this change. It makes it impossible to tell where any hex types come from.

Other than that, ACK this.

Reply:
Just to clarify, you mean you dislike the wildcard, right? The re-export is ok?

Reply:
I also question re-export of `buf_encoder`. The `display::*` items should be mostly sufficient. We should probably document that things inside `buf_encoder` are low-level details.

Reply:
Added to the top of `buf_encoder.rs`
```
//! This is a low-level module, most uses should be satisfied by the `display` module instead.
```

Reply:
And remove the re-export of `buf_encoder` stuff.

Reply:
hex/src/lib.rs is now:
```rust
pub use display::{DisplayHex, DisplayArray};
pub use parse::{Error, FromHex, HexIterator};
```